### ISSUE TITLE
Blinking on and off

### ISSUE THREAD

**Issue by oshh94** [AlexGustafsson/homebridge-wol#18]
Hello.

For some reason it be blinking will going on/off constantly. It dosen't send the WOL request, but still responds to when i press on. This is purely "cosmetic". This only happens with NAS1, my second NAS dosen't have this problem.

log

Apr 23 20:33:41 raspberrypi homebridge[1366]: [4/23/2017, 8:33:41 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:33:42 raspberrypi homebridge[1366]: [4/23/2017, 8:33:42 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:33:46 raspberrypi homebridge[1366]: [4/23/2017, 8:33:46 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:33:47 raspberrypi homebridge[1366]: [4/23/2017, 8:33:47 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:33:51 raspberrypi homebridge[1366]: [4/23/2017, 8:33:51 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:33:52 raspberrypi homebridge[1366]: [4/23/2017, 8:33:52 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:33:56 raspberrypi homebridge[1366]: [4/23/2017, 8:33:56 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:33:57 raspberrypi homebridge[1366]: [4/23/2017, 8:33:57 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:34:01 raspberrypi homebridge[1366]: [4/23/2017, 8:34:01 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:34:02 raspberrypi homebridge[1366]: [4/23/2017, 8:34:02 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:34:06 raspberrypi homebridge[1366]: [4/23/2017, 8:34:06 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:34:07 raspberrypi homebridge[1366]: [4/23/2017, 8:34:07 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:34:11 raspberrypi homebridge[1366]: [4/23/2017, 8:34:11 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:34:12 raspberrypi homebridge[1366]: [4/23/2017, 8:34:12 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:34:16 raspberrypi homebridge[1366]: [4/23/2017, 8:34:16 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:34:17 raspberrypi homebridge[1366]: [4/23/2017, 8:34:17 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:34:21 raspberrypi homebridge[1366]: [4/23/2017, 8:34:21 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:34:22 raspberrypi homebridge[1366]: [4/23/2017, 8:34:22 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:34:26 raspberrypi homebridge[1366]: [4/23/2017, 8:34:26 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:34:27 raspberrypi homebridge[1366]: [4/23/2017, 8:34:27 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:34:31 raspberrypi homebridge[1366]: [4/23/2017, 8:34:31 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:34:32 raspberrypi homebridge[1366]: [4/23/2017, 8:34:32 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:34:36 raspberrypi homebridge[1366]: [4/23/2017, 8:34:36 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:34:37 raspberrypi homebridge[1366]: [4/23/2017, 8:34:37 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:34:41 raspberrypi homebridge[1366]: [4/23/2017, 8:34:41 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:34:42 raspberrypi homebridge[1366]: [4/23/2017, 8:34:42 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:34:46 raspberrypi homebridge[1366]: [4/23/2017, 8:34:46 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:34:47 raspberrypi homebridge[1366]: [4/23/2017, 8:34:47 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:34:51 raspberrypi homebridge[1366]: [4/23/2017, 8:34:51 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:34:52 raspberrypi homebridge[1366]: [4/23/2017, 8:34:52 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:34:56 raspberrypi homebridge[1366]: [4/23/2017, 8:34:56 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:34:57 raspberrypi homebridge[1366]: [4/23/2017, 8:34:57 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:35:01 raspberrypi homebridge[1366]: [4/23/2017, 8:35:01 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:35:02 raspberrypi homebridge[1366]: [4/23/2017, 8:35:02 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:35:06 raspberrypi homebridge[1366]: [4/23/2017, 8:35:06 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:35:07 raspberrypi homebridge[1366]: [4/23/2017, 8:35:07 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:35:11 raspberrypi homebridge[1366]: [4/23/2017, 8:35:11 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:35:12 raspberrypi homebridge[1366]: [4/23/2017, 8:35:12 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:35:16 raspberrypi homebridge[1366]: [4/23/2017, 8:35:16 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:35:17 raspberrypi homebridge[1366]: [4/23/2017, 8:35:17 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:35:21 raspberrypi homebridge[1366]: [4/23/2017, 8:35:21 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:35:22 raspberrypi homebridge[1366]: [4/23/2017, 8:35:22 PM] [NAS1] NAS1(192.168.200.200) was just turned on
Apr 23 20:35:26 raspberrypi homebridge[1366]: [4/23/2017, 8:35:26 PM] [NAS1] NAS1(192.168.200.200) was just turned off
Apr 23 20:35:27 raspberrypi homebridge[1366]: [4/23/2017, 8:35:27 PM] [NAS1] NAS1(192.168.200.200) was just turned on

config.json
`{
    "bridge": {
        "name": "Homebridge",
        "username": "CC:22:3D:E3:CE:30",
        "port": 51826,
        "pin": "031-45-154"
    },

    "description": "This is an example configuration file with one fake accessory and one fake platform. You can use this as a template for creating your own configuration file containing devices you actually own.",

    "accessories": [{
            "accessory": "Computer",
            "name": "NAS2",
            "mac": "24:5E:BE:0D:79:3B",
            "ip": "192.168.200.202"
        },
        {
            "accessory": "Computer",
            "name": "NAS",
            "mac": "00:08:9B:CA:21:82",
            "ip": "192.168.200.200"
        },
        {
            "accessory": "Computer",
            "name": "Mediacenter1",
            "mac": "1C:B7:2C:B1:6F:D3",
            "ip": "192.168.200.210"
        },
        {
            "accessory": "PiTemperature",
            "name": "Raspberry PI Temperature"
        }
    ],

    "platforms": []
}`

root@raspberrypi:/home/pi# node -v
v6.10.2
root@raspberrypi:/home/pi# npm -v
3.10.10

Linux raspberrypi 4.4.50+ #970 Mon Feb 20 19:12:50 GMT 2017 armv6l GNU/Linux

**Comment by AlexGustafsson** [comment]
I tink that you're completely right. I've tried my best to look into the issue, but I think it's an IOS thing. During my tests, if I flicked the switch and then updated the screen on the device (whether it was Control Center or the Home app) the behaviour was as expected.

I believe it has to do with how IOS expects there to actually be some change in the state when the switch is flicked, and rather than checking with the server, the UI shows the switch in the wanted state for user experience reasons.
